packages/wallet/src/redux/protocols/close-ledger-channel/states.ts

```diff
@@ -0,0 +1,71 @@
+import { WithdrawalState } from '../withdrawing/states';
+import { AdvanceChannelState } from '../advance-channel';
+import { StateConstructor } from '../../utils';
+import { ProtocolState } from '..';
+
+export interface WaitForWithdrawal {
+  type: 'CloseLedgerChannel.WaitForWithdrawal';
+  processId: string;
+  withdrawal: WithdrawalState;
+  channelId;
+}
+export interface WaitForConclude {
+  type: 'CloseLedgerChannel.WaitForConclude';
+  processId: string;
+  concluding: AdvanceChannelState;
+  channelId;
+}
+
+export interface Failure {
+  type: 'CloseLedgerChannel.Failure';
+  reason: string;
```

Comment:
I've moved away from doing this just for ease of development and the fact that we don't use the reason yet.

It might be good for future proofing so I'll go through and update them.

Reply:
I think it has these benefits:
- it serves as good documentation
- it enables auto-complete
- it will be useful when specifying how protocols should handle failure